games/game.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../items'
+
+class Game < Item
+  attr_accessor :multiplayer, :last_played_date
+
+  def initialize(name, publish_date, multiplayer, last_played_date)
+    super(name, publish_date)
+    @multiplayer = multiplayer
+    @last_played_date = last_played_date
+  end
+
+  def to_s
+    "Game: #{@name}, #{@publish_date}, #{@multiplayer}, #{@last_played_date}"
+  end
+
+  def can_be_archived?
+    current_date = Date.today
+    year = current_date.year - @publish_date.year
+    return true if year > 2
```

Comment:
you have to change this part of the code, like the requirement says it suppost to be:should return true if parent's method returns true AND if last_played_at is older than 2 years something like 

`super && year>2`